mmcv/runner/base_module.py

```diff
@@ -26,6 +43,14 @@ def __init__(self, init_cfg=None):
         self._is_init = False
         self.init_cfg = init_cfg
 
+        # The `params_init_info` is used to record the initialization
+        # information of the parameters
+        # the key should be the obj:`nn.Parameter` of model and the value
+        # should be a dict contains
+        # `params_name`, `init_info` and `tmp_sum_value`.
```

Comment:
- What does those keys mean? The comments should specify that.
- Why "params_name"? Does it indicate more than one parameters?